controllers/authorino_controller.go

```diff
@@ -0,0 +1,598 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	"github.com/go-logr/logr"
+	authorinooperatorv1beta1 "github.com/kuadrant/authorino-operator/api/v1beta1"
```

Comment:
I'd be fine with either

```suggestion
	operator "github.com/kuadrant/authorino-operator/api/v1beta1"
```

or 

```suggestion
	api "github.com/kuadrant/authorino-operator/api/v1beta1"
```

If it doesn't specify further, then it's ours; otherwise we make it more explicit, such as `k8sapps "k8s.io/api/apps/v1"`, `k8score "k8s.io/api/core/v1"`, etc.

WDYT?